glean-core/src/upload/request.rs

```diff
@@ -194,8 +203,9 @@ impl PingRequest {
     ///
     /// * `language_binding_name` - The name of the language used by the binding that instantiated this Glean instance.
     ///                             This is used to build the User-Agent header value.
-    pub fn builder(language_binding_name: &str) -> Builder {
-        Builder::new(language_binding_name)
+    /// * `body_max_size` - The maximum size the compressed ping body may have to be eligible for upload.
```

Comment:
Ah, the parameter is documented here (because that's how we use the builder).
Still should say `in bytes`